doc/team.rst

```diff
@@ -0,0 +1,91 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+We are an international team dedicated to building a Pythonic API for the
+Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
+new and experienced users by creating user-friendly interfaces with the GMT
+C API, supporting rich display in Jupyter notebooks, and integrating with
+the PyData ecosystem.
+
+Maintainers
+-----------
+
+.. raw:: html
+
+    <div class="team-row">
+    <div class="team-column">
+    <div class="team-card">
+        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/3974108?v=4" alt="Card image">
+        <div class="team-card-body">
+            <h6 class="team-card-text">Dongdong Tian</h6>
+            <p class="team-card-text"><a href="https://github.com/seisman" >@seisman</a></p>
+        </div>
+    </div>
+    </div>
+    <div class="team-column">
+    <div class="team-card">
+        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/14077947?v=4" alt="Card image">
+        <div class="team-card-body">
+            <h6 class="team-card-text">Meghan Jones</h6>
+            <p class="team-card-text"><a href="https://github.com/meghanrjones" >@meghanrjones</a></p>
+        </div>
+    </div>
+    </div>
+    <div class="team-column">
+    <div class="team-card">
+        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/23025878?v=4" alt="Card image">
+        <div class="team-card-body">
+            <h6 class="team-card-text">Michael Grund</h6>
+            <p class="team-card-text"><a href="https://github.com/michaelgrund" >@michaelgrund</a></p>
+        </div>
+    </div>
+    </div>
+    <div class="team-column">
+    <div class="team-card">
+        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/23487320?v=4" alt="Card image">
+        <div class="team-card-body">
+            <h6 class="team-card-text">Wei Ji Leong</h6>
+            <p class="team-card-text"><a href="https://github.com/weiji14" >@weiji14</a></p>
+        </div>
+    </div>
+    </div>
+    <div class="team-column">
+    <div class="team-card">
+        <img class="card-img-top team-img" src="https://avatars.githubusercontent.com/u/29518865?v=4" alt="Card image">
+        <div class="team-card-body">
+            <h6 class="team-card-text";>Will Schlitzer</h6>
+            <p class="team-card-text"><a href="https://github.com/willschlitzer" >@willschlitzer</a></p>
+        </div>
+    </div>
+    </div>
+    </div>
+    <p></p>
+
+Emeritus Maintainers
+--------------------
```

Comment:
[Emeritus](https://en.wikipedia.org/wiki/Emeritus) sounds a bit old for these two young fellas, is there a better term maybe?

Reply:
Oops, my bad. Maybe 'Former Maintainers'? Or we could combine the sections with some design choice to indicate activate maintainers.